packages/ui/src/components/Footer/index.tsx

```diff
@@ -99,26 +100,28 @@ export const Footer = ({ data }: FooterProps) => (
           </Grid>
         </GridCell>
         <GridCell md={4} sm={2} className={css('utrecht-grid--justify-content-sm-flex-end')}>
-          <ul className={css('utrecht-link-list', 'utrecht-link-list--html-ul', 'utrecht-link-list--social-media')}>
-            {data?.socialMediaList &&
-              data.socialMediaList?.link.length > 0 &&
-              data.socialMediaList?.link.map((item, index) => {
-                const Icon = socialMediaIconTypes[item.icon as keyof typeof socialMediaIconTypes];
-                return (
-                  <li key={index} className={css('utrecht-link-list__item')}>
-                    <Link
-                      external
-                      href={item.href}
-                      className={css('utrecht-link-list__link')}
-                      aria-label={item.textContent}
-                    >
-                      <Icon />
-                      <span className="utrecht-link-list__link-text">{item.textContent}</span>
-                    </Link>
-                  </li>
-                );
-              })}
-          </ul>
+          <nav aria-label={socialMediaAriaLabel}>
```

Comment:
Liever niet aria-label, maar aria-labelledby. Is het een goed idee om even een eigen `<Nav>` component te bouwen, die werkt zoals in de BreadcrumbNav? Dat 'ie automatisch een `Heading level={levle}` ofzo maakt met een label erin?